bundles/org.openhab.binding.exec/src/main/java/org/openhab/binding/exec/internal/handler/ExecHandler.java

```diff
@@ -289,7 +288,7 @@ public void execute() {
 
             outputBuilder.append(errorBuilder.toString());
 
-            String transformedResponse = StringUtils.chomp(outputBuilder.toString());
+            String transformedResponse = outputBuilder.toString().replaceAll("\r", "").replaceAll("\n", "");
```

Comment:
This is not the correct equivalent. According to https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#chomp-java.lang.String- only the last `\n`, `\r`, `\r\n` is removed.

Reply:
See also https://github.com/openhab/openhab-addons/pull/14433#discussion_r1111050883

Reply:
The tests i used was wrong, got it fixed now including a util class with tests.

Reply:
This got replaced by a core util class.